web/src/settings_config.ts

```diff
@@ -513,6 +513,48 @@ export const expires_in_values = {
     },
 };
 
+export const realm_deletion_in_values = {
+    immediately: {
+        value: 0,
+        description: $t({defaultMessage: "Immediately"}),
+        default: false,
+    },
+    fourteen_days: {
+        value: 14 * 24 * 60,
+        description: $t({defaultMessage: "14 days"}),
+        default: false,
+    },
+    thirty_days: {
+        value: 30 * 24 * 60,
+        description: $t({defaultMessage: "30 days"}),
+        default: false,
+    },
+    ninty_days: {
+        value: 90 * 24 * 60,
+        description: $t({defaultMessage: "90 days"}),
+        default: false,
+    },
+    one_year: {
+        value: 365 * 24 * 60,
+        description: $t({defaultMessage: "1 Year"}),
+        default: false,
+    },
```

Comment:
Added a "2 years" option and fixed the capitalization for "1 year" to match the other options.